### PR TITLE
Alk slitasje

### PR DESCRIPTION
**Pull request by anders-kolstad**
Please review this half finished indicator called _slitasje_.

### CODE REVIEW

**Review by HannoSandvik — COMMENTED** [review]
I have read until line 190.

**Comment by anders-kolstad** [comment]
After the [last commit](https://github.com/NINAnor/ecosystemCondition/pull/14/commits/5a6a210c8cfe08ee0ca919b64a301ede3bef0d8c) the chaper renders locally. The final export stages are not yet coded. This is awaiting eaTools development.

File: DRAFT_slitasje.Rmd [hunk 1]
```diff
@@ -0,0 +1,1496 @@
...
+For a run through of the data set, see [here](#naturtype).
+
+## Ecosystem characteristic
+
+### Norwegain standard
+
+The indicator is tagged to the *Økologisk egenskap* called
+**Primærproduksjon** (Primary productivity). This is tentative, and
+perhaps *abiotiske forhold* is more suited. But the thought behind the
+chiose is that *sliatsje* affect the potential for primary productivity.
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:124]: "choice"

File: DRAFT_slitasje.Rmd [hunk 2]
```diff
@@ -0,0 +1,1496 @@
...
+
+### UN standard
+
+The indicator is tagged as a **Physical state characteristics**
+indicator. This is quite clear. It's a tpe of abiotic characteritic that
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:129]: "characteristic".

"type"

File: DRAFT_slitasje.Rmd [hunk 3]
```diff
@@ -0,0 +1,1496 @@
...
+has to do with the physical structure rather than the chemical
+composition.
+
+## Collinearities with other indicators
+
+Slitasje is not thought to be very collinear with any other indicator at
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:135]: "... thought to exhibit collinearity with any other indicator.."

File: DRAFT_slitasje.Rmd [hunk 4]
```diff
@@ -0,0 +1,1496 @@
...
+
+The reference condition is one with minimal negative human impact. This
+is also true for semi-natural ecosystems. For example, in the reference
+condition, *slitasje* in semi-natural ecosystems is defined to be zero,
+even though at no point in time would this condition be realized.
+
+### Reference values, thresholds for defining *good ecological condition*, minimum and/or maximum values
+
+The upper reference value is 0 (no *slitasje*), and the lower reference
+value is 100%. Note that 100% slitasje doesn't mean that all the area
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:150]: "does not" (doesn't is "spoken English" written!)

File: DRAFT_slitasje.Rmd [hunk 5]
```diff
@@ -0,0 +1,1496 @@
...
+must be scared or damaged, but that all hypothetical quadrats around a
+point is affected (the variable is frequency-driven, and not
+amount-driven). The threshold for good ecosystem condition is 10%
+damage. This is on hte conservative side. Perhaps is should be set
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:154]: "...on the..."

File: DRAFT_slitasje.Rmd [hunk 6]
```diff
@@ -0,0 +1,1496 @@
...
+somewhat higher, like 20 or 30%.
+
+Read about the normalisation [here](##scaled-slitasje-variable).
+
+## Uncertainties
+
+Uncertaintiesare estimated for aggregated indicator values by
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:161]: "Uncertainties are"

File: DRAFT_slitasje.Rmd [hunk 7]
```diff
@@ -0,0 +1,1496 @@
...
+See also [here](#naturtype) for a detailed description of the data set.
+
+<!-- Importing the full data set can be done like this, but because it can -->
+
+<!-- take a few seconds due to its size, I will cache the data on the P -->
+
+<!-- server after subsetting. -->
+
+```{r, eval=F}
+naturetypes <- sf::st_read(dsn = paste0(pData, "Natur_Naturtyper_NiN_norge_med_svalbard_25833.gdb"))
```
Comment (DrMattG) [inline DRAFT_slitasje.Rmd:193]: I cant access P!

Comment (DrMattG) [reply DRAFT_slitasje.Rmd:193]: So will stop here for now

**Review by DrMattG — CHANGES_REQUESTED** [review]

File: DRAFT_slitasje.Rmd [hunk 8]
```diff
@@ -0,0 +1,1496 @@
...
+
+## Introduction
+
+1.  Import [Nature type data](#NTM) data set (incl. GRUK) and [ANO data
+    set](#ANO-import)
+
+2.  Identify the [relevant](#naturtype) nature types and [subset](#NTM)
+    the data
+
+3.  Covert [ANO points to polygons](#ANO-points-to-poly)
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:61]: "Convert"

File: DRAFT_slitasje.Rmd [hunk 9]
```diff
@@ -0,0 +1,1496 @@
...
+
+7.  Define [homogeneous impact areas](#HIA) (HIA)
+
+8.  [Aggregate and spread](#spread-slitasje) indicator value across HIAs
+    and assessment regions
+
+9.  Confirm relationship between infrastructure index and indicator
+    values to justify the extrapolation
+
+10. Sample 10 00 random points from an ecosystem delineation and extract
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:79]: remove space in "1000"

File: DRAFT_slitasje.Rmd [hunk 10]
```diff
@@ -0,0 +1,1496 @@
...
+
+11. Export indicator maps and regional extrapolated maps
+
+## About the underlying data
+
+The indicator uses a data set from a standardised field survey of nature
+types. You can read more about the preliminary analyses
+[here](#naturtype). See also the [official
+site](https://www.miljodirektoratet.no/ansvarsomrader/overvaking-arealplanlegging/naturkartlegging/naturtyper/)
+of the Environmental Agency. I also import a data set called
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:90]: officially the "[Norwegian] Environment Agency" without -al)

File: DRAFT_slitasje.Rmd [hunk 11]
```diff
@@ -0,0 +1,1496 @@
...
+```{r, eval=F}
+saveRDS(naturetypes, "/data/P-Prosjekter2/41201785_okologisk_tilstand_2022_2023/data/rds/naturtyper.rds")
+```
+
+```{r, include=F}
+# Import back in
+naturetypes <- readRDS(paste0(pData, "rds/naturtyper.rds"))
+```
+
+```{r, fig.cap="An overview of the naturetypes for which we will calculate the indicator. Colours refer to the main ecosystem affiliation."}
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:246]: A colour legend might be nice.

File: DRAFT_slitasje.Rmd [hunk 12]
```diff
@@ -0,0 +1,1496 @@
...
+library(knitr)
+library(sf)
+library(tmap)
+library(ggplot2)
+library(ggpubr)
+library(DT)
+library(units)
+library(eaTools)
+library(tidyverse)
+library(tidyr)
```
[the remark this answers is not in the document]
Comment (anders-kolstad) [reply DRAFT_slitasje.Rmd:24]: done

File: DRAFT_slitasje.Rmd (hunk 11 above)
Comment (anders-kolstad) [reply DRAFT_slitasje.Rmd:246]: added

File: DRAFT_slitasje.Rmd (hunk 5 above)
Comment (anders-kolstad) [reply DRAFT_slitasje.Rmd:154]: ok

File: DRAFT_slitasje.Rmd (hunk 4 above)
Comment (anders-kolstad) [reply DRAFT_slitasje.Rmd:150]: very good

File: DRAFT_slitasje.Rmd (hunk 3 above)
Comment (anders-kolstad) [reply DRAFT_slitasje.Rmd:135]: good

File: DRAFT_slitasjeOgKjørespor.Rmd [hunk 13]
```diff
@@ -46,82 +40,105 @@ metaData <- data.frame(Ecosystem,
...
 #### Nature type mapping
-This indicator uses the dataset `Naturtyper etter Miljødirektoratets Instruks`, which can be found [here](https://kartkatalog.geonorge.no/metadata/naturtyper-miljoedirektoratets-instruks/eb48dd19-03da-41e1-afd9-7ebc3079265c).
-The dataset has already been [cleaned](#naturtype1) and [initial analyses on thematic representativity](#naturtype2) has been done.
 
-```{r, eval=T}
-dat <- readRDS("/data/P-Prosjekter2/41201785_okologisk_tilstand_2022_2023/data/Mdir_Naturtyper_cleanedALK.rds")
+This indicator uses the dataset
+`Naturtyper etter Miljødirektoratets Instruks`, which can be found
+[here](https://kartkatalog.geonorge.no/metadata/naturtyper-miljoedirektoratets-instruks/eb48dd19-03da-41e1-afd9-7ebc3079265c).
+See [here](#naturtype) for a detailed description of the dataset.
```
[the remark this answers is not in the document]
Comment (anders-kolstad) [reply DRAFT_slitasjeOgKjørespor.Rmd]: Hopefully these will work when we tie it with the rest of the book

File: DRAFT_slitasje.Rmd [hunk 14]
```diff
@@ -0,0 +1,1496 @@
...
+naturetypes <- naturetypes[!is.na(naturetypes$slitasje),]
+```
+
+#### GRUK
+
+This variable is also recorded in
+[GRUK](https://www.nina.no/Naturmangfold/Trua-natur/%C3%85pen-grunnlendt-kalkmark).
+The nature type data set I'm working on here includes this data already
+(presently only 2021 included). GRUK also records a related variable:
+`% cover in 5m radii circles`, which si much more detailed. This data is
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:506]: "which _is_ much more"

File: DRAFT_slitasje.Rmd [hunk 15]
```diff
@@ -0,0 +1,1496 @@
...
+not published. In any case it is better to use the harmonized data set
+in our case.
+
+#### ANO {#ANO-import}
+
+Arealrepresentativ Naturovervåking (ANO) consist of 1000 systematically
+placed locations each with 18 sample points. In each sample point a
+circle of 250 m2 is visualised, and the main ecosystem is recorded.
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:514]: probably not "visualised" (_synliggjort_) but "mapped" or "inspected"?

Comment (anders-kolstad) [reply DRAFT_slitasje.Rmd:514]: Visualised I think is correct. It is a hypothetical and imagined circle.

**Review by DrMattG — APPROVED** [review]

File: DRAFT_slitasje.Rmd [hunk 16]
```diff
@@ -0,0 +1,1497 @@
...
+proportion of impacted areas. This is expected.
+
+### Aggregate and spread (extrapolate) {#spread-slitasje}
+
+I now want to find the mean indicator value for each HIA (i.e. to
+aggregate) and to spread these out spatially to the entire HIA within
+each region (i.e. to extrapolate).
+
+I want to add a threshold so that we don't end up over extrapolating
+based on too few data points. From Fig. \@ref(fig:dense-stats) I
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:1224]: The figure link in this line doesn't seem to work

File: DRAFT_slitasje.Rmd [hunk 17]
```diff
@@ -0,0 +1,1497 @@
...
+For the finest resolution indicator map I will not include columns for
+uncertainty (because this will only be estimated at the aggregated
+level), or reference levels (because there is no spatial variation).
+
+But I will also export a map of the indicator where we extrapolate the
+indicator values to the homogeneous areas. This map should be
+interpreted with caution, since they seem to provide explicit indicator
+values for areas where there is no data. This map is mainly for
+aggregation purposes. I will mark this map with the prefix
+*homogeneous-areas*. I can also export this same mappen after masking it
```
Comment (HannoSandvik) [inline DRAFT_slitasje.Rmd:1460]: "map" (not "mappen") + (next line:) "disclaimers"

**Review by HannoSandvik — COMMENTED** [review]
I have now read the second half of the slitasje chapter and made a few comments.

**Pull request merged**